session/session.go

```diff
@@ -18,14 +18,13 @@ package session
 
 import (
 	"fmt"
+	"github.com/softlayer/softlayer-go/config"
+	"github.com/softlayer/softlayer-go/sl"
```

Comment:
Thank you. But I think the two dependencies are from the upstream repo code.

https://github.com/softlayer/softlayer-go/blob/master/session/session.go#L27-L28